src/pagination.rs

```diff
@@ -545,3 +568,20 @@ fn interpret_permalink(
         permalink::explode_permalink(&config.permalink, &attributes)?
     })
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn compare_category_path_test() {
+        let a = vec![liquid::value::Value::scalar("A")];
+        let b = vec![liquid::value::Value::scalar("B")];
+        let ar = liquid::value::Value::array(a);
+        let ar = ar.as_array().unwrap();
+        let br = liquid::value::Value::array(b);
```

Comment:
btw liquid now has a macro for creating stuff in a cleaner manner.  Not too many examples of it but https://github.com/cobalt-org/liquid-rust/blob/2b01636c754a70248b1ee61b43b8369ba502414c/liquid-value/src/values.rs#L588

Reply:
Thanks!

Reply:
How to use it outside of liquid.rs? I'm seeing it being defined in liquid.rs' `macros.rs` which is not `pub` in `lib.rs`

Reply:
Did you try a `macro_use` on `liquid_value`?

Reply:
I tried like this:
```rust
#[cfg(test)]
mod test {
    use super::*;

    #[macro_use]
    use liquid::liquid_value;

    #[test]
    fn compare_category_path_test() {
      let ar = liquid_value!(["A"]);
        let br = liquid_value!(["B"]);
        assert_eq!(Ordering::Less, compare_category_path(&ar, &br));
        assert_eq!(Ordering::Greater, compare_category_path(&br, &ar));
    }
}
```

and in `lib.rs` as 
```rust
#[macro_use]
extern crate liquid;
``` 
with no success

Reply:
I don't even know if this test is worth it, it was for me to test the comparison function but it's trivial enough I think. Though?

Reply:
Either way.  I don't think using the macros should be blocking though.  Thats something we can fix separately from this.